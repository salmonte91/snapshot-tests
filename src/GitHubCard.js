import Alert from 'react-bootstrap/Alert'

function GitHubCard() {
  return (
    <Alert variant='success'>
        <p>
            If you are reading this, then I have successfully made the alert!
        </p>
    </Alert>
  )
}

export default GitHubCard