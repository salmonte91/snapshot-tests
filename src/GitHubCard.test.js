import renderer from 'react-test-renderer'
import GitHubCard from './GitHubCard'

test('render a snapshot', () => {
    const tree = renderer.create(<GitHubCard/>).toJSON()
    expect(tree).toMatchSnapshot()
  })