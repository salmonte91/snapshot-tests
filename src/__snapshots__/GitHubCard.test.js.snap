// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a snapshot 1`] = `
<div
  className="fade alert alert-success show"
  role="alert"
>
  <p>
    If you are reading this, then I have successfully made the alert!
  </p>
</div>
`;
